backend/src/test/java/gov/cdc/usds/simplereport/test_util/JsonTestUtils.java

```diff
@@ -66,6 +66,7 @@ public static void assertJsonNodesEqual(
       if (expectedFieldValue.isObject() && actualFieldValue.isObject()) {
         assertJsonNodesEqual(expectedFieldValue, actualFieldValue, ignoredFields, fieldPath);
       } else if (expectedFieldValue.isArray() && actualFieldValue.isArray()) {
+        assertThat(expectedFieldValue.size()).isEqualTo(actualFieldValue.size());
```

Comment:
Good catch here for when the actual array might have more items than the expected!

Reply:
👍 yeah I wanted to make sure it will catch if a field is unexpectedly added since we've had that happen before and our tests didn't catch it. it also forces us to keep the expected value bundles up to date when we add new entries since it will now fail if they're not